### PR TITLE
Decoupling addon from Ember Observer API

### PR DESCRIPTION
**Pull request by gmurphey**
After discussing with @kategengler in #6, we think this is the better solution rather than relying on the private Ember Observer API.

### CODE REVIEW

File: src/get-addon-metadata.js [hunk 1]
```diff
@@ -24,22 +24,22 @@ function promisedRequest(url) {
...
+    addon = addon.replace(/\@|\/|\./g, '-')
+    return promisedRequest(`https://emberobserver.com/badges/${addon}.svg`);
   });
 
   let responses = await Promise.all(requests);
-
-  let successfulResponses = responses.filter((response) => {
-    return !response.errors;
-  })
+  let successfulResponses = responses;
```
Comment (kategengler) [inline src/get-addon-metadata.js]: Unnecessary assignment

Comment (gmurphey) [reply src/get-addon-metadata.js]: Thanks! Reminds me I need to get linting set up.

Comment (gmurphey) [reply src/get-addon-metadata.js]: Thanks! Reminds me I need to get linting set up.

**Comment by kategengler** [comment]
Thanks for switching to this -- the images are batch updated and won't go beyond nginx in emberobserver infrastructure and it is much less likely to be changed.

**Pull request merged**